securedrop/manage.py

```diff
@@ -15,6 +17,8 @@
 
 from db import db_session, Journalist
 
+from management.run import run
```

Comment:
@jacksingleton It'd be nice to make this say `from management import run`, but I don't think there's a way to do that while also maintaining a separate `management/run.py` file, which is nice for encapsulation. Do you have any ideas?